bin/test-run

```diff
@@ -16,11 +16,11 @@ init_test_run "$@"
 printf '==================RUNNING ALL TESTS==================\n'
 printf 'Testing Linkerd version [%s] namespace [%s] k8s-context [%s]\n' "$linkerd_version" "$linkerd_namespace" "$k8s_context"
 
-upgrade_integration_tests
-helm_integration_tests
+#upgrade_integration_tests
+#helm_integration_tests
 helm_upgrade_integration_tests
-deep_integration_tests
-external_issuer_integration_tests
+#deep_integration_tests
+#external_issuer_integration_tests
```

Comment:
Look like you forgot to uncomment :wink:

Reply:
thanks for catching that!